packages/contract/lib/execute.js

```diff
@@ -208,18 +225,12 @@ const execute = {
             contract: constructor
           });
 
-          const stacktrace = promiEvent.debug ? promiEvent.debug : false;
-          try {
-            params.gas = await execute.getGasEstimate.call(
-              constructor,
-              params,
-              network.blockLimit,
-              stacktrace
-            );
-          } catch (error) {
-            promiEvent.reject(error);
-            return;
-          }
+          params.gas = await execute.getGasEstimate.call(
```

Comment:
Do we not need to reject the `promiEvent` any more? I'm not sure offhand how exactly this should work. If it throws here it will just stop all execution so the `promiEvent` shouldn't matter any more?

Reply:
As best I can tell, simply allowing this to throw works fine.  Doing this didn't break any tests, after all.  I was not able to discern any difference between throwing and having the `promiEvent` reject.

Reply:
Oh, duh -- the reason it makes no difference is because the whole thing is wrapped in `.catch(promiEvent.reject)`!

Reply:
oh haha, so it will reject anyways...cool - this is confusing

Reply:
Yeah :-/